dist/src/main/java/io/camunda/application/commons/migration/MigrationsRunner.java

```diff
@@ -34,7 +34,20 @@ public MigrationsRunner(final List<Migrator> migrators) {
   @Override
   public void run(final ApplicationArguments args) throws Exception {
     LOG.info("Starting {} migration tasks", migrators.size());
-    final Exception migrationsExceptions = new Exception("migration is failed");
+    /* Detach from main Spring thread */
+    new Thread(
```

Comment:
:x: I still think we should set a thread name here.

:wrench: Ideally add one too for the executor used to run the actual migrations.